.github/actions/tune-runner-vm/action.yml

```diff
@@ -0,0 +1,12 @@
+name: Tune Runner VM
+description: tunes the GitHub Runner VM operation system
```

Comment:
Please be more specific in name, file name and description and mention `reverse lookups for current hostname`

Reply:
I added a better description of the intention of the "tune-runner-vm".